### DISCUSSION TITLE
Open files by unique ID

### DISCUSSION THREAD

**Discussion by timkrueger** [helix-editor/helix#8402] [Q&A] [answered]
Hey all. 

I'm relative new to Helix. I want to use it as editor for my note system (Zettelkasten) in which I work with linked files. A short example.

All my notes are written in Markdown. All files in are in a single directory:

```
.
├── 202201151253 A note.md
├── ...
└── 202201171534 Anoter note.md
```

The files can be linked via their ID:

```markdown
# A note

A link to another note: [[202201171534]]
```

Is there a way to open `202201171534 Anoter note.md` if the cursor is over the ID in `202201151253 A note.md`?

**Comment by David-Else** [comment, marked as answer]
Have you tried https://github.com/artempyanykh/marksman ? It is the default language server for Markdown.

**Reply by timkrueger** [reply]
Thank you @David-Else . I didn't know about that and it looks like it will work for me. 🙏🏻
